README.md

```diff
@@ -34,10 +44,198 @@ Then grab a copy of the private key or mnemonic registered to your Farcaster use
 
 ## Examples
 
-### Publish a Cast
+### Neynar API
```

Comment:
Would be helpful to add to this README instructions for other contributors:
- how to update the openapi code
- how to run the tests (any secrets needed)
- how to structure commits (commitizen)
- anything else you came across

At some point we'll need separate READMEs for this module and the REST-client module. Can happen either in this PR or later when I put up a PR for splitting out the separate packages